### ISSUE TITLE
Problem with capital or small letters

### ISSUE THREAD

**Issue by bflorat** [jajuk-team/jajuk#1957]
**Reported by manfred on 9 Jan 2014 17:38 UTC**
Jajuk shoud not make Differences between capital or small letters.
Many Artists are listed double ore more

**Comment by gkerdal** [comment]
I try to use MD5Processor.hash(sName.trim().toLowerCase()); locally to see the behaviour on a new fresh collection of 2 albums of the same artist with 2 different cases : let's say Group1 and group1.
1- The collection view shows only one artist, good point. The display name is group1.
2 - Let's say I'm not satisfied by the lower case choice, I'd prefer Goup1. I try to update the property value : I enter and validate "Group1" but then nothing happens. If Jajuk relies on the id, it's not a bug since Id(group1)=Id(Group1)
3 - If I change group1 by group12, then the update of the artist name is processed. I get the expected behaviour

In summary, the default artist name may not be the user's choice so we should provide a way to correct the case easily.
